ui/panels-plugin/src/plugins/gauge-chart/index.ts

```diff
@@ -12,3 +12,4 @@
 // limitations under the License.
 
 export * from './GaugeChart';
+export type { GaugeChartDefinition } from './gauge-chart-model';
```

Comment:
I decided to be specific / explicit with the export here, but I'm cool with doing `*` if anyone thinks that's better

Reply:
I think it's fine to start more focused. It's much easier to add things in the future if we want to expose them than to take them away if we realize the * export exposed a feature we didn't want to support externally.